src/main/python/fine_tuning/evaluation.py

```diff
@@ -62,13 +62,13 @@ def output_all_evaluations(self, qrel_programs, qrel_file_path, result_file_path
             if process.returncode == 0:
                 try:
                     if i == 0:
-                        o = open( output_path, 'w')
+                        o = open(output_path, 'w')
                     else:
-                        o = open( output_path, 'a')
+                        o = open(output_path, 'a')
                     if 'trec_eval' in qrel_program:
-                        o.write(stdout)
+                        o.write(stdout.decode("utf-8"))
```

Comment:
Why we need a `decode` here?

Reply:
If I don't have `decode` there, I get the following error:

```
"""
Traceback (most recent call last):
  File "/anaconda3/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/anaconda3/lib/python3.7/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "src/main/python/fine_tuning/run_batch.py", line 112, in atom_eval
    Evaluation.output_all_evaluations(*params)
  File "/Users/jimmylin/workspace/Anserini/src/main/python/fine_tuning/evaluation.py", line 69, in output_all_evaluations
    o.write(stdout)
TypeError: write() argument must be str, not bytes
"""
```

Reply:
interesting -- might be a python2 vs python3 compatibility issue.